packages/@aws-cdk/aws-rds/lib/serverless-cluster.ts

```diff
@@ -99,11 +99,13 @@ interface ServerlessClusterNewProps {
 
   /**
    * The VPC that this Aurora Serverless cluster has been created in.
+   *
+   * @default - no VPC will be used
    */
-  readonly vpc: ec2.IVpc;
+  readonly vpc?: ec2.IVpc;
 
   /**
-   * Where to place the instances within the VPC
+   * Where to place the instances within the VPC. If set, a VPC must be specified.
```

Comment:
Let's mult-line this:

```suggestion
   * Where to place the instances within the VPC.
   * If provided, the `vpc` property must also be specified.
```